docs/getting-started/cases-req.asciidoc

```diff
@@ -1,6 +1,11 @@
 [[case-permissions]]
 = Cases prerequisites
 
+:frontmatter-description: Learn about the {kib} feature privileges required to access {elastic-sec} cases. 
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to] 
+:frontmatter-tags-user-goals: [configure]
+
```

Comment:
I wasn't aware that we're adding frontmatter to ESS doc files. Is this something we should be doing for all files?

Reply:
Probably a good idea to double-check with @colleenmcginnis to be sure, but it was originally something that helped fill in the frontmatter for migration.